packages/devtools_app/lib/src/shared/table.dart

```diff
@@ -1133,45 +1129,45 @@ class TableRow<T> extends StatefulWidget {
   final bool isShown;
 
   /// The children to show when the expand animation of this widget is running.
-  final List<Widget> expansionChildren;
-  final VoidCallback onExpansionCompleted;
+  final List<Widget>? expansionChildren;
+  final VoidCallback? onExpansionCompleted;
 
   /// The background color of the row.
   ///
   /// If null, defaults to `Theme.of(context).canvasColor`.
-  final Color backgroundColor;
+  final Color? backgroundColor;
 
-  final ColumnData<T> sortColumn;
+  final ColumnData<T>? sortColumn;
 
-  final SortDirection sortDirection;
+  final SortDirection? sortDirection;
```

Comment:
both of these should be non nullable

Reply:
nope, they are null by default in the constructor